src/layout.js

```diff
@@ -207,7 +208,7 @@ export function hasNaturalDimensions(tagName) {
  */
 export function getNaturalDimensions(tagName) {
   tagName = tagName.toUpperCase();
-  if (!naturalDimensions_[tagName]) {
+  if (naturalDimensions_[tagName] === null) {
```

Comment:
Both are somewhat wrong. A better way is:
```
assert(naturalDimensions_[tagName] !== undefined);
if (!naturalDimensions_[tagName]) {
...
```

Assert must be there since the API doesn't allow one to calculate natural dimensions just for any element. Once assert is done, it doesn't matter if we do `!x` or `x === null`, but `!x` is cleaner.